.travis.yml

```diff
@@ -76,4 +76,4 @@ script:
   - if [ "$TEST_SUITE" = "accessibility" ]; then docker exec -it social_ci_web bash /var/www/scripts/social/check-accessibility.sh install; fi
 
 after_success:
-  - bash scripts/social/trigger_dockerhub.sh
+  - if [ "$TRAVIS_PULL_REQUEST_SLUG" == "goalgorilla/open_social" ]; then bash scripts/social/trigger_dockerhub.sh
```

Comment:
Make sure you end your statement correctly so travis doesnt end 

Also see why you have to do it like below:
https://www.thegeekstuff.com/2010/06/bash-if-statement-examples

`if [ "$TRAVIS_PULL_REQUEST_SLUG" == "goalgorilla/open_social" ]; then bash scripts/social/trigger_dockerhub.sh; fi`

Reply:
Thank you @ronaldtebrake. This one I might have overlooked when committed even I copied and paste in. However, there is a new commit on way.